libs/components/action-bars/src/lib/modules/summary-action-bar/summary-action-bar.component.html

```diff
@@ -7,6 +7,29 @@
   }"
 >
   <div class="sky-summary-action-bar-actions">
+    <div aria-live="polite">
+      @if (formErrors) {
```

Comment:
Can we combine these two if statements to `@if (formErrors?.length)`?

Reply:
i did have that initially but it threw build errors 😭